integration/prune_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// This file is ignored during the regular tests due to the following build tag.
+//go:build rpctest
+// +build rpctest
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/integration/rpctest"
+)
+
+func TestPrune(t *testing.T) {
+	t.Parallel()
+
+	// Boilerplate code to make a pruned node.
+	btcdCfg := []string{"--prune=1536"}
+	r, err := rpctest.New(&chaincfg.SimNetParams, nil, btcdCfg, "")
+	if err != nil {
+		t.Fatal("unable to create primary harness: ", err)
```

Comment:
Should we use the `require` library for new code? Makes things more compact IMO.